index.html

```diff
@@ -55,6 +55,162 @@ <h1>I’m Mariane Glad to see you!</h1>
                 </li>
             </ul>
         </section>
+
+        <div class="my-projects">
+            <div class="project">
+                <img src="images/project1.png" alt="project1 icon">
+                <p class="project-title">Tonic</p>
+                <div class="project-sub-header">
+                    <span>CANOPY</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">Back End Dev</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">2015</span>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#" class="project-details">See Project</a>
+            </div>
+            <div class="project">
+                <img src="images/project2.png" alt="project2 icon">
+                <p class="project-title">Multi-Post Stories</p>
+                <div class="project-sub-header">
+                    <span>CANOPY</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">Back End Dev</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">2015</span>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#" class="project-details">See Project</a>
+            </div>
+            <div class="project">
+                <img src="images/project3.png" alt="project3 icon">
+                <p class="project-title">Tonic</p>
```

Comment:
- [ ] For this heading, replace this and the rest with a `<hx>` tag. This is so as to enhance your semantics.